ArcGISRuntimeSDKQt_CppSamples/Features/FilterByDefinitionExpressionOrDisplayFilter/FilterByDefinitionExpressionOrDisplayFilter.cpp

```diff
@@ -133,3 +133,32 @@ void FilterByDefinitionExpressionOrDisplayFilter::queryFeatureCountInCurrentExte
 
   m_featureTable->queryFeatureCount(parameters);
 }
+
+void FilterByDefinitionExpressionOrDisplayFilter::resetDisplayFilterParams()
+{
+  DisplayFilter* displayFilter = new DisplayFilter("Damaged Trees", "", this);
+  const QList<DisplayFilter*> available_filters{displayFilter};
+
+  ManualDisplayFilterDefinition* display_filter_defintion = new ManualDisplayFilterDefinition(displayFilter, available_filters, this);
+  m_featureLayer->setDisplayFilterDefinition(display_filter_defintion);
+
+  queryFeatureCountInCurrentExtent();
+}
+
+void FilterByDefinitionExpressionOrDisplayFilter::resetDefExpressionParams()
+{
+  m_featureLayer->setDefinitionExpression("");
+
+  connect(m_mapView, &MapQuickView::drawStatusChanged, this, [this](DrawStatus drawStatus)
+  {
+    drawStatus == DrawStatus::InProgress ? m_mapDrawing = true : m_mapDrawing = false;
+    emit mapDrawStatusChanged();
+
+    queryFeatureCountInCurrentExtent();
```

Comment:
Is there a reason `queryFeatureCountInCurrentExtent()` should run when the map is still drawing?
```suggestion
    if (drawStatus == DrawStatus::Completed)
    {
      m_mapDrawing = false;
      queryFeatureCountInCurrentExtent();
    }
    else
    {
      m_mapDrawing = true;
    }
    
    emit mapDrawStatusChanged();
```